WalletWasabi.Fluent/ViewModels/Login/LoginViewModel.cs

```diff
@@ -54,7 +54,7 @@ private async Task OnNextAsync(ClosedWalletViewModel closedWalletViewModel, Wall
 		if (!isPasswordCorrect)
 		{
 			IsForgotPasswordVisible = true;
-			ErrorMessage = "The password is incorrect! Try Again.";
+			ErrorMessage = "The password is incorrect! Try again.";
```

Comment:
```suggestion
			ErrorMessage = "The password is incorrect! Please try again.";
```
Consistent with above